core/src/util/usFrameworkPrivate.cpp

```diff
@@ -168,6 +169,8 @@ void FrameworkPrivate::Shutdown(bool restart)
   case Bundle::STATE_STOPPING:
     // Shutdown already inprogress
     break;
+  default:
```

Comment:
My point is, that we are already checking this at compile time. If an enumeration value would not be handled, the compiler throws an error.